### PR TITLE
ci: output log when install criu

### PR DESCRIPTION
**Pull request by Ace-Tang**
ci test alway hang on installing criu things some times ago, add
log to make sure who blocked the ci, see #2298.

Signed-off-by: Ace-Tang <aceapril@126.com>

### Ⅲ. Why don't you add test cases (unit test/integration test)? (你真的觉得不需要加测试吗？)

no need.

### CODE REVIEW

**Comment by fuweid** [comment]
interesting. the change is LGTM but I didn't see any output in the CI log. 🤔

**Comment by fuweid** [comment]
my bad. Please remove the redirection.

```
criu::ubuntu::install_dependencies > /dev/null
```

**Comment by Ace-Tang** [comment]
Keep redirection, echo log before function, or log will too many, I did not consider this before.

File: hack/install/install_criu.sh [hunk 1]
```diff
@@ -48,6 +48,7 @@ main() {
   os_dist="$(detect_os)"
   if [[ "${os_dist}" = "Ubuntu" ]]; then
     criu::ubuntu::install_dependencies > /dev/null
+    echo ">>>> start to download criu from github repository <<<<"
```
Comment (allencloud) [inline hack/install/install_criu.sh]: If the downloading task fails, would this script show the detailed reason why it fails? @Ace-Tang

File: hack/install/install_criu.sh [hunk 2]
```diff
@@ -48,6 +48,7 @@ main() {
   os_dist="$(detect_os)"
   if [[ "${os_dist}" = "Ubuntu" ]]; then
```
Comment (fuweid) [inline hack/install/install_criu.sh]: could we add one comment here?

File: hack/install/install_criu.sh (hunk 1 above)
Comment (fuweid) [reply hack/install/install_criu.sh]: I think yes.

Comment (Ace-Tang) [reply hack/install/install_criu.sh]: only know we stack at downing from github.

File: hack/install/install_criu.sh (hunk 2 above)
Comment (Ace-Tang) [reply hack/install/install_criu.sh]: I do not think so, `>>>> install criu <<<<` will tell begin to download dependencies. and `>>>> start to download criu from github repository <<<<` will tell begin to download from github. Then we can know which step stack.

File: hack/install/install_criu.sh (hunk 1 above)
Comment (allencloud) [reply hack/install/install_criu.sh]: ok, then it meets my demand.

**Pull request merged**